packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -300,9 +300,9 @@ class Dropdown extends React.PureComponent {
                                     /> :
                                     Object.keys(this.props.list).map(key => (
                                         <React.Fragment key={key}>
-                                            <div className='dc-list__label'>{key}</div>
+                                            <div className={classNames('dc-list__label', this.props.classNameLabel)}>{key}</div>
                                             <Items
-                                                index={this.state.curr_index}
```

Comment:
it is used to highlight items, but it's not working correctly for object of objects, since the item index gets reset and curr_index will be found twice, so it highlights two items or however many objects you have inside your object. so I'm removing it and in the future if someone passes object of objects to autocomplete and needs the highlight feature they can figure out how to fix highlighting items there :D